src/features/customerList/CustomerListContainer.tsx

```diff
@@ -65,6 +64,20 @@ const orderBySelector = selector<string | undefined>({
   },
 });
 
+// Backend supports queries for names in the form of firstName, lastName only so
+// this transforms SearchBy.NAME to firstName, lastName.
+const transformSearchByName = (searchBy: SearchBy, value: string) => {
```

Comment:
Works in most of the cases, but what when the name is in 3 parts (or more)? Should the lenght be checked first? If the lenght is 3 or more, which parts belongs to first and which to the lastname?

Reply:
Now it's pretty stupid, but I was thinking since the first input is always the last name, it might be ok for now to just let admins type enough of the last name to get close enough to meaningful results (since results update while typing), and then use first name to "refine" the search. Now anything after second part is just ignored. 

So essentially now the problems would come from two part (without hyphen) last names - IF the admins are fast enough to type both parts - , and last names with some kind of prefix like "Af".. Maybe the small prefixes might be good idea to include with first part though - that could cause issues actually.

Reply:
We discussed this with the team and came to the conclusion that this is good enough for now for most common use cases, but we'll keep the last name and first name options for more exotic names (added on the next commit: https://github.com/City-of-Helsinki/berth-reservations-admin/pull/490/commits/35db2b64fa9fccaba6ad1cc9ad20f51651b1b2a5)